specs/Standards/Tokens/FungibleToken.md

```diff
@@ -48,6 +59,11 @@ There are a few concepts in the scenarios above:
 Note, that the precision is not part of the default standard, since it's not required to perform actions. The minimum
 value is always 1 token.
 
+The standard acknowledges NEAR storage staking model and accounts for the difference in storage that can be introduced
+by actions on this contract. Since multiple users use the contract, the contract has to account for potential
+storage increase. That's why every change method of the contract that can change the amount of storage has to be payable.
```

Comment:
```suggestion
storage increase. Thus every change method of the contract that can change the amount of storage must be payable.
```

Just curious, why can't this cost be covered by gas?  Since it's variable?

Reply:
Also if a method only decreases state why would an attached deposit be needed?

Reply:
It's required to be part of the account balance. Since the gas can only be used to write into the storage, but not to permanently store it.